pkg/input/golden_test/tf/null-count.json

```diff
@@ -8,8 +8,8 @@
   },
   "resources": {
     "aws_s3_bucket": {
-      "aws_s3_bucket.foo": {
-        "id": "aws_s3_bucket.foo",
+      "aws_s3_bucket.foo[0]": {
```

Comment:
I don't think it should block this PR, but does this behavior make sense? Terraform errors when `count = null`, which wouldn't be appropriate for us, but maybe we should be treating it as `0` rather than `1`?

Reply:
added a comment in https://github.com/snyk/policy-engine/pull/154/commits/f3b7e5042dcf840e771271b3c65863e7ca84e1b7